src/test/scala/com/typesafe/webdriver/HtmlUnitWebDriverCommandsSpec.scala

```diff
@@ -32,7 +31,7 @@ class HtmlUnitWebDriverCommandsSpec extends Specification with NoDurationConvers
   def testFor(v: JsValue): MatchResult[Any] = {
     val result = withSession {
       (commands, sessionId) =>
-        commands.executeJs(sessionId, "var result = arguments[0];", JsArray(v))
+        commands.executeJs(sessionId._1, "var result = arguments[0];", JsArray(v))
```

Comment:
`sessionId` is the value passed in by `withSession` (defined just above `testFor`). This value is the result from `commands.createSession()` where `commands` is an instance of `WebDriverCommands`.
In 94b158a I changed the signature of `createSession()` from `Future[String]` to `Future[(String, Either[WebDriverError, JsValue])]` where `String` is the session id and `Either[WebDriverError, JsValue]` is either the unavailable capabilities error or a JSON object listing the actual capabilities of the session. 
This change allows me to either report the actual session capabilities to the user of the library or notify it that the session could not be created properly (see https://github.com/jeantil/webdriver/commit/93b60244d08bc9dc0429425e8f5ce6635b51a1c2#diff-e24d239250304c137be2113d1397b132R30)